pallets/executive/src/lib.rs

```diff
@@ -329,7 +333,17 @@ where
             // execute extrinsics
             let (header, extrinsics) = block.deconstruct();
 
-            Self::execute_extrinsics_with_book_keeping(extrinsics, *header.number());
+            let extrinsics_with_author: Vec<(Option<_>,_)> = info.into_iter().zip(extrinsics.into_iter()).collect();
```

Comment:
Am I understand right the tx authorship info can not be accessed here inside executive using only data from `block`?

Reply:
yep, only in this scope we have full information about all the types, inside ExecutivePallet these are not available